### PR TITLE
[NLP-166] Add calibrated probability scores to classifier model

### PR DESCRIPTION
**Pull request by boyangdong00**
1. Reproduce new pickle file for calibrated complementNB.
2. Removed old function that manually updated confidence value in front end 
3. Update make_model function for retraining
4. Tested front end and back end for partial fit and prediction.
5. Updated notebook in training for reproduction of the calibrated pipeline

### CODE REVIEW

**Review by JCayabyab — APPROVED** [review]
Frontend changes look good to me. Please wait for Calum's review before merging

**Review by rynoV — CHANGES_REQUESTED** [review]
Works on my end. Before merging this, let's wait for #86 and make sure the model retraining works with this new pipeline.

**Pull request merged**
